packages/js/generators.json

```diff
@@ -7,14 +7,14 @@
       "schema": "./src/generators/library/schema.json",
       "aliases": ["lib"],
       "x-type": "library",
-      "description": "Create a library"
+      "description": "Create a library."
     },
     "convert-to-swc": {
       "factory": "./src/generators/convert-to-swc/convert-to-swc#convertToSwcSchematic",
       "schema": "./src/generators/convert-to-swc/schema.json",
       "aliases": ["swc"],
       "x-type": "library",
-      "description": "Convert a tsc library to swc"
+      "description": "Convert a Typescript library to SWC."
```

Comment:
It's still a TypeScript library, just different compiler.

Something like this is probably better: `Convert a TypeScript library to compile with SWC`